### PR TITLE
Adds in a jetpack to the EVA crate while increasing the cost. New crate that costs 3,000 and includes a Integrated jetpack

### PR DESCRIPTION
**Pull request by ma44**
As title says

Discuss about balance and stuff below

### CODE REVIEW

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -484,17 +484,27 @@
...
-					/obj/item/clothing/mask/breath)
+					/obj/item/clothing/mask/breath,
+					/obj/item/weapon/tank/jetpack/oxygen)
 	crate_name = "space suit crate"
 	crate_type = /obj/structure/closet/crate/secure
-
+	
+/datum/supply_pack/engineering/engine/spacesuitupgrade
+	name = "Hardsuit Integrated Jetpack upgrade"
+	cost = 3000
```
Comment (Jalleo) [inline code/modules/cargo/packs.dm]: I would say double it.

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -484,17 +484,27 @@
 
 /datum/supply_pack/engineering/engine/spacesuit
 	name = "Space Suit Crate"
-	cost = 3000
+	cost = 4000
```
Comment (Jalleo) [inline code/modules/cargo/packs.dm]: Eh.

File: code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -484,17 +484,27 @@
...
 	access = access_eva
 	contains = list(/obj/item/clothing/suit/space,
 					/obj/item/clothing/suit/space,
 					/obj/item/clothing/head/helmet/space,
 					/obj/item/clothing/head/helmet/space,
 					/obj/item/clothing/mask/breath,
-					/obj/item/clothing/mask/breath)
+					/obj/item/clothing/mask/breath,
+					/obj/item/weapon/tank/jetpack/oxygen)
```
Comment (Jalleo) [inline code/modules/cargo/packs.dm]: Not CO2

**Comment by ma44** [comment]
@Jalleo im pretty damm sure thats an oxygen tank
https://github.com/tgstation/tgstation/blob/685fc1a91c6a3fc51618dcbe8f5ce7b2496e1e60/code/game/objects/items/weapons/tanks/jetpack.dm#L83

**Comment by ma44** [comment]
slightly increased the integrated jackpack price, waiting for more people to say its op or something and needs a price increase

**Comment by KorPhaeron** [comment]
Sorry, we have to fix the infinite points every round aspect of cargo before we throw more junk into it

**Pull request closed without merging**